projects/solid/src/index.ts

```diff
@@ -0,0 +1,44 @@
+import {
+    Maskito,
+    MASKITO_DEFAULT_ELEMENT_PREDICATE,
+    type MaskitoElementPredicate,
+    type MaskitoOptions,
+} from '@maskito/core';
+import {
+    createEffect,
+    createResource,
+    createSignal,
+    onCleanup,
+    type Setter,
+} from 'solid-js';
+
+export const useMaskito = ({
+    options = null,
+    elementPredicate = MASKITO_DEFAULT_ELEMENT_PREDICATE,
+}: {
+    options?: MaskitoOptions | null;
+    elementPredicate?: MaskitoElementPredicate;
+} = {}): Setter<HTMLElement | null> => {
```

Comment:
Does `Soild` has `directive` concept ?
If yes, let's use it (see `@maskito/angular` & `@maskito/vue` as examples).

I am not sure but probably `Solid` has it:
https://docs.solidjs.com/reference/jsx-attributes/use#use

Reply:
```jsx
import {maskito} from '@maskito/solid';
// [...]
<input type="text" use:maskito={[options, predicate]} />
```

Reply:
If directive is an option we should definitely go for it.